docs/StardustDocs/topics/schemasGradle.md

```diff
@@ -150,60 +150,3 @@ print(df.fullName.count { it.contains("kotlin") })
 ```
 
 <!---END-->
-
```

Comment:
Hm? What happened to this?

Reply:
I forgot to remove it after extracting the content to the new page